app/models/page.rb

```diff
@@ -60,10 +60,9 @@ def get_binding
     end
 
     def render(*args)
-      av = ActionView::Base.new
-      av.view_paths = ActionController::Base.view_paths
-      av.view_paths << Rails.root.join("app/views/pages").to_s
-      av.render(*args)
+      ActionController::Base.append_view_path "app/views/pages"
+      renderer = ActionController::Base.renderer.new
+      renderer.render(partial: args.first)
```

Comment:
@matthewd What do you think about the change here? 🙇‍♂️

Reply:
The first line is mutating a global value, but this method gets called multiple times.

Sounds to me like the `append_view_path` should be in `PagesController`, then this would be:

```ruby
def render(partial)
  PagesController.render(partial: partial)
end
```

Reply:
Thanks! Fixed in 4488f93.